### PR TITLE
Updated broken links

### PR DESCRIPTION
**Pull request by sushreebarsa**
A few broken links and typos were updated in documentation.

# Description

> :memo: Please include a summary of the change. 
>  
> * Please also include relevant motivation and context.  
> * List any dependencies that are required for this change.  

## Type of change

For a new feature or function, please create an issue first to discuss it
with us before submitting a pull request.

Note: Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Documentation update
- [ ] TensorFlow 2 migration
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] A new research paper code implementation
- [ ] Other (Specify)

## Tests

> :memo: Please describe the tests that you ran to verify your changes.
>  
> * Provide instructions so we can reproduce.  
> * Please also list any relevant details for your test configuration.  

**Test Configuration**:

## Checklist

- [ ] I have signed the [Contributor License Agreement](https://github.com/tensorflow/models/wiki/Contributor-License-Agreements).
- [ ] I have read [guidelines for pull request](https://github.com/tensorflow/models/wiki/Submitting-a-pull-request).
- [ ] My code follows the [coding guidelines](https://github.com/tensorflow/models/wiki/Coding-guidelines).
- [ ] I have performed a self [code review](https://github.com/tensorflow/models/wiki/Code-review) of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.

### CODE REVIEW

**Review by yeqingli — DISMISSED** [review]

File: official/README-TPU.md [hunk 1]
```diff
@@ -5,25 +5,25 @@
...
     See [Tensorboard.dev training metrics](https://tensorboard.dev/experiment/KnaWjrq5TXGfv0NW5m7rpg/#scalars).
-*   [mnist](vision/image_classification): A basic model to classify digits
+*   [mnist](https://www.tensorflow.org/api_docs/python/tf/keras/datasets/mnist): A basic model to classify digits
     from the MNIST dataset. See [Running MNIST on Cloud TPU](https://cloud.google.com/tpu/docs/tutorials/mnist-2.x) tutorial and [Tensorboard.dev metrics](https://tensorboard.dev/experiment/mIah5lppTASvrHqWrdr6NA).
-*   [mask-rcnn](vision/detection): An object detection and instance segmentation model. See [Tensorboard.dev training metrics](https://tensorboard.dev/experiment/LH7k0fMsRwqUAcE09o9kPA).
-*   [resnet](vision/image_classification): A deep residual network that can
+*   [mask-rcnn](https://www.tensorflow.org/api_docs/python/tfm/vision/configs/maskrcnn/MaskRCNN): An object detection and instance segmentation model. See [Tensorboard.dev training metrics](https://tensorboard.dev/experiment/LH7k0fMsRwqUAcE09o9kPA).
+*   [resnet](https://www.tensorflow.org/api_docs/python/tf/keras/applications/resnet): A deep residual network that can
```
Comment (yeqingli) [inline official/README-TPU.md:22]: Should be `https://www.tensorflow.org/api_docs/python/tfm/vision/configs/image_classification/image_classification_imagenet`.

Comment (sushreebarsa) [reply official/README-TPU.md]: I have made the changes as required. Thank you!

**Comment by gbaned** [comment]
Hi @yeqingli Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @yeqingli Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @Chen-Zhe Can you please review this PR ? Thank you!

**Comment by laxmareddyp** [comment]
Closing this PR as these changes submitted through [internal CL](https://github.com/tensorflow/models/commit/8e9f80a27e4abd2ed889701cd34d42ef2a6e9373)

**Pull request closed without merging**